### PR TITLE
fix(e2e_tests): broken tests following build_096 integration

### PR DESCRIPTION
**Pull request by Millefeuille42**
#6046

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

~- [ ] Manual, quality tests have been done~ Do not need tests for tests.

### CODE REVIEW

File: functionnal_tests/cypress/integration/activity_feed/activity_spec.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ describe('At the space recent activities page', () => {
       cy.get('[data-cy=commentArea__comment__send]').click()
     })
     it('should display the file preview', () => {
+      cy.wait(2000)
```
Comment (Skylsmoi) [inline functionnal_tests/cypress/integration/activity_feed/activity_spec.js:50]: ow
it hurts : (
we'll have to at it

Comment (Millefeuille42) [reply functionnal_tests/cypress/integration/activity_feed/activity_spec.js:50]: Yes, I'm currently working on it! (and noticed I shouldn't do this anymore ^^')

Comment (Skylsmoi) [reply functionnal_tests/cypress/integration/activity_feed/activity_spec.js:50]: ok ^^
I can help you if you need, I'll be way more available starting from today

File: functionnal_tests/cypress/integration/activity_feed/activity_spec.js [hunk 2]
```diff
@@ -38,18 +38,18 @@ describe('At the space recent activities page', () => {
...
       cy.get('[data-cy=popup__createcontent__form__button]').click()
       cy.get('[data-cy=commentArea__comment__send]').click()
-    })
-    it('should display the file preview', () => {
-      cy.wait(2000)
-      cy.get('[data-cy=FilenameWithBadges__label]').first().should('include.text', 'News')
-      cy.get('.CommentFilePreview > .attachedFile').should('be.visible')
+      cy.get('.attachedFile')
+      cy.visitPage({ pageName: PAGES.RECENT_ACTIVITIES, params: { workspaceId }, waitForTlm: true })
+      cy.get('.CommentFilePreview > .attachedFile')
```
Comment (gbonaspetti) [inline functionnal_tests/cypress/integration/activity_feed/activity_spec.js:52]:
```suggestion
      cy.get('.attachedFile').should('be.visible')
      cy.visitPage({ pageName: PAGES.RECENT_ACTIVITIES, params: { workspaceId }, waitForTlm: true })
      cy.get('.CommentFilePreview > .attachedFile').should('be.visible')
```

**Review by gbonaspetti — APPROVED** [review]

File: functionnal_tests/cypress/integration/activity_feed/activity_spec.js (hunk 2 above)
Comment (Millefeuille42) [reply functionnal_tests/cypress/integration/activity_feed/activity_spec.js]: We were discussing this with @Skylsmoi yesterday, doesn't `cy.get` already implicitly tests `should(be.visible)` ? If so, did you suggested it to make the code more explicit ?

**Pull request merged**